src/EditorFeatures/Core/Implementation/Suggestions/SuggestedAction.cs

```diff
@@ -116,8 +117,14 @@ protected SolutionPreviewResult GetPreviewResult(CancellationToken cancellationT
         {
             if (_operations == null)
             {
-                _operations = Task.Run(
+                var extensionManager = this.Workspace.Services.GetService<IExtensionManager>();
```

Comment:
GetPreviewChangesSuggestedActionSet does not use extensonManager.PerformFunction.  I'll add this there.